qiita_pet/webserver.py

```diff
@@ -57,6 +57,16 @@
     ReloadPluginAPItestHandler)
 from qiita_pet import uimodules
 from qiita_db.util import get_mountpoint
+from qiita_pet.handlers.rest import (
+    StudyHandler as WeDontKnowWhatToCallThis,
```

Comment:
What about prefixing all the handlers with API or REST? It will avoid collisions with anything else and it will be clear that is part of the rest api